core/src/packets/ip/v6/mod.rs

```diff
@@ -419,7 +405,7 @@ pub const IPV6_PACKET: [u8; 78] = [
     0x20, 0x01, 0x0d, 0xb8, 0x85, 0xa3, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01,
     // dst addr
     0x20, 0x01, 0x0d, 0xb8, 0x85, 0xa3, 0x00, 0x00, 0x00, 0x00, 0x8a, 0x2e, 0x03, 0x70, 0x73, 0x34,
-    // ** TCP header
+// TCP header
```

Comment:
aligning in these parts seem weird no?

Reply:
on purpose, marks where each header section starts and standout from the regular comments with no indentation. before I was use `// **`, didn't really standout and it's weird af.

Reply:
yeah, still reads a bit weird, but ok.